testdata/local/pod_dispatchers.json

```diff
@@ -0,0 +1 @@
+{"_id":"dispatcher0", "pod_ids":  ["localhost"]}
```

Comment:
Do we need this in the local data? Local Evergreen typically doesn't have data to run tasks.

Reply:
I was just putting a dispatcher in the local DB in case we'd need it for some debugging purpose in the future but since it's not needed today I can omit it.